gaia-header.js

```diff
@@ -278,9 +278,9 @@ module.exports = component.register('gaia-header', {
   setTitleStyle: function(el, style) {
     debug('set title style', style);
     this.observerStop();
-    el.style.MozMarginStart = style.marginStart + 'px';
-    el.style.MozPaddingStart = style.padding.start + 'px';
-    el.style.MozPaddingEnd = style.padding.end + 'px';
+    el.style.marginLeft = style.marginStart + 'px';
+    el.style.paddingLeft = style.padding.start + 'px';
+    el.style.paddingRight = style.padding.end + 'px';
```

Comment:
and the very fact that the spec does not patch gecko's implementation makes it clear we should not use it except for cases where the element's directionality is the same than the parent's directionality.